client/client.go

```diff
@@ -49,11 +49,7 @@ type Client struct {
 	hub *hub.Hub
 }
 
-func New(configPath, driver string, dsn string) (*Client, error) {
-	cfg, err := config.Parse(configPath)
-	if err != nil {
-		return nil, err
-	}
+func New(driver, dsn string, cfg *config.Config) (*Client, error) {
```

Comment:
Pass in a reference to a config object instead of a file path to parse. This is the main change. It requires users of the client to create a config outside of the client initialization